tests/test_config.py

```diff
@@ -244,6 +244,16 @@ def testOperators(self):
         c2[".a.b"] = 5
         self.assertNotEqual(c1, c2)
 
+    def testMerge(self):
+        c1 = Config({"a": 1, "c": 2})
+        c2 = Config({"a": 3, "b": "b"})
```

Comment:
Having single-letter values which match the single letter key names (especially later when mapping "b" to "c") means that one has to read the code more carefully.   It would just make the reading of this test easier if the string values were either small words or the other end of the alphabet.